memorywipe.py

```diff
@@ -138,36 +162,41 @@ def _veracrypt_interactive(self):
         }
         filesystem = filesystem_dict[click.prompt("Select", default=2, type=click.IntRange(1, 6), show_choices=False)]
         click.echo("\nEnter strong password for encrypting. (You don't have to remember it)")
-        strongp = click.prompt("So set a random password with special characters: ", hide_input=True, confirmation_prompt=False)
-        out = subprocess.run(["sudo", "veracrypt", "-tc", f"--volume-type={vol_type}", 
-                                f"{self.partition}", f'--encryption={encrypt_type}', f'--hash={hash_algo}', 
-                                f"--filesystem={filesystem}", "-p", f"{strongp}", "--pim=0", 
-                                "-k", "", "--random-source=/dev/urandom"], capture_output=True)
+        strongp = click.prompt("So set a random password with special characters: ", hide_input=True,
+                               confirmation_prompt=False)
+        out = subprocess.run(["sudo", "veracrypt", "-tc", f"--volume-type={vol_type}",
+                              f"{self.partition}", f'--encryption={encrypt_type}', f'--hash={hash_algo}',
+                              f"--filesystem={filesystem}", "-p", f"{strongp}", "--pim=0",
+                              "-k", "", "--random-source=/dev/urandom"], capture_output=True)
         if out.returncode:
             click.secho(out.stderr.decode(), fg="yellow")
-        show_out = click.prompt("Do you wish to see the output of the command run?", type=click.Choice(["y", "n"]), default="n")
+        show_out = click.prompt("Do you wish to see the output of the command run?", type=click.Choice(["y", "n"]),
```

Comment:
Is it possible to remove the choice here, and simply show the output? I think it will be better to show the output.

Reply:
Isn't the veracrypt volume creation output lengthy?  It may exceed beyond the terminal screen and hence i added it as a choice. If that's not an issue, then i'll remove it right away

Reply:
Let's do one thing, send me a screenshot of the output. We can decide if it is required to hide it or not.

Reply:
Sure. The output is a small "volume successfully created " message. I'll push the changes with no output prompt.

I actually faced an issue while sanitising. It may be a bug perhaps that during wiping and mounting device, my wifi driver service process was terminated. It turned on back after the process got completed. I am not sure this might occur in other devices as well.

Reply:
Is this happening every time?

Reply:
> Is this happening every time?

No. Out of 7-8 time sanitization tests I performed on the USB stick, It occured only once.

Reply:
Then it might be something else. But if it repeats again, let me know